tastyworks/models/order.py

```diff
@@ -33,10 +34,17 @@ def is_active(self):
         return self in (OrderStatus.LIVE, OrderStatus.RECEIVED)
 
 
+class TimeInForce(Enum):
+    DAY = 'Day'
+    GTC = 'GTC'
+    GTD = 'GTD'
+
+
 @dataclass
 class OrderDetails(object):
     type: OrderType = None
-    time_in_force: str = 'Day'
+    time_in_force: TimeInForce = TimeInForce.DAY
+    gtc_date: str = None
```

Comment:
Could we store this as a `datetime`? I think it would make a lot more sense to API users if they don't have to parse the string themselves.

Reply:
👍 Good call, updated.